src/fractalrhomb_globals.py

```diff
@@ -89,7 +93,7 @@ async def save(self, fp: str) -> None:
 			discord_logger.info("Saved bot data.")
 
 
-bot_data = BotData({}, {}, {})
+bot_data = BotData({}, {}, {}, {})
```

Comment:
I'm gonna test it to make sure nothing's fucked, but real quick, the second one should be a `[]` since the type is a list now.